tests/e2e/steps_sovereign_changeover.go

```diff
@@ -0,0 +1,353 @@
+				chainA:  chainID(consumerName),
+				chainB:  chainID("provi"),
+				clientA: 1,
+				clientB: 1,
+			},
+			state: State{},
+		},
+		{
+			action: addIbcChannelAction{
+				chainA:      chainID(consumerName),
+				chainB:      chainID("provi"),
+				connectionA: 1,
+				portA:       "consumer",
+				portB:       "provider",
+				order:       "ordered",
+			},
+			state: State{},
+		},
+	}
+
+	return s
+}
+
+// start sovereign chain with 2 validators
+// nodes will cease being validators once the changeover occurs
+func stepRunSovereignChain() []Step {
+	return []Step{
+		{
+			action: StartSovereignChainAction{
+				chain: chainID("sover"),
+				validators: []StartChainValidator{
+					{id: validatorID("alice"), stake: 500000000, allocation: 10000000000},
+				},
+			},
+			state: State{
+				chainID("sover"): ChainState{
+					ValBalances: &map[validatorID]uint{
+						validatorID("alice"): 9500000000,
+					},
+				},
+			},
+		},
+		{
+			action: delegateTokensAction{
+				chain:  chainID("sover"),
+				from:   validatorID("alice"),
+				to:     validatorID("alice"),
+				amount: 11000000,
+			},
+			state: State{
+				chainID("sover"): ChainState{
+					ValPowers: &map[validatorID]uint{
+						validatorID("alice"): 511,
+						validatorID("bob"):   0, // does not exist on pre-ccv sover
+						validatorID("carol"): 0, // does not exist on pre-ccv sover
+					},
+				},
+			},
+		},
+	}
+}
+
+// TODO: use args instead of hardcoding
+func stepsUpgradeChain() []Step {
+	return []Step{
+		{
+			action: UpgradeProposalAction{
+				chainID:       chainID("sover"),
+				upgradeTitle:  "sovereign-changeover",
+				proposer:      validatorID("alice"),
+				upgradeHeight: 110,
+			},
+			state: State{
+				chainID("sover"): ChainState{
+					Proposals: &map[uint]Proposal{
+						1: UpgradeProposal{
+							Title:         "sovereign-changeover",
+							UpgradeHeight: 110,
+							Type:          "/cosmos.upgrade.v1beta1.SoftwareUpgradeProposal",
+							Deposit:       10000000,
+							Status:        "PROPOSAL_STATUS_VOTING_PERIOD",
+						},
+					},
+				},
+			},
+		},
+		{
+			action: voteGovProposalAction{
+				chain:      chainID("sover"),
+				from:       []validatorID{validatorID("alice")},
+				vote:       []string{"yes"},
+				propNumber: 1,
+			},
+			state: State{
+				chainID("sover"): ChainState{
+					Proposals: &map[uint]Proposal{
+						1: UpgradeProposal{
+							Deposit:       10000000,
+							UpgradeHeight: 110,
+							Title:         "sovereign-changeover",
+							Type:          "/cosmos.upgrade.v1beta1.SoftwareUpgradeProposal",
+							Status:        "PROPOSAL_STATUS_PASSED",
+						},
+					},
+				},
+			},
+		},
+		{
+			action: waitUntilBlockAction{
+				chain: chainID("sover"),
+				block: 110,
+			},
+			state: State{},
+		},
+	}
+}
+
+// stepsPostChangeoverDelegate tests basic delegation and resulting validator power changes
+func stepsPostChangeoverDelegate(consumerName string) []Step {
```

Comment:
Comment why you cannot reuse stepsDelegate

Reply:
Done